### PR TITLE
doc: make lifecycle images have a transparent background

### PR DESCRIPTION
**Pull request by paulidale**
- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by t8m — APPROVED** [review]

File: doc/life-cycles/cipher.dot [hunk 1]
```diff
@@ -1,4 +1,6 @@
 digraph cipher {
+    bgcolor="#00000000";
```
Comment (levitte) [inline doc/life-cycles/cipher.dot:2]: https://graphviz.org/doc/info/attrs.html#k:color

Try `bgcolor = "transparent"`?

Comment (paulidale) [reply doc/life-cycles/cipher.dot:2]: That's the same end result as setting the alpha channel to zero.  I guess being explicit is better.

**Review by t8m — APPROVED** [review]

File: doc/life-cycles/cipher.dot (hunk 1 above)
Comment (paulidale) [reply doc/life-cycles/cipher.dot]: The output image files are identical with this change.

**Review by levitte — APPROVED** [review]
Yes!

**Comment by paulidale** [comment]
Merged, thanks for the suggestions.

**Pull request closed without merging**